src/corona_hakab_model/analyzers/simulation_analysis.py

```diff
@@ -0,0 +1,24 @@
+import os
+from glob import glob
+from project_structure import OUTPUT_FOLDER
+import pandas as pd
+from datetime import datetime
+
+
+def create_comparison_files(parameters_to_compare: list, files: list = None):
```

Comment:
Use the headers from the CSV as columns

Reply:
How is the new solution? Using set and columns I get all the columns (Notice that this IS affected by casing, but I am not sure if I should deal with it for now, I cna check)